.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+yarn lint-staged
```

Comment:
Just noticed the husky pre-commit was not working. It was simple change to add to this PR